app/javascript/stylesheets/admin_form.scss

```diff
@@ -0,0 +1,49 @@
+.form-component {
+  width: 50%;
+  display: block;
+  margin: auto;
+  padding: 20px;
+  background-color: rgba(255, 255, 255, 0.4);
```

Comment:
duplicated color, move common colors to https://github.com/IF-115-Ruby/OSBB/blob/development/app/javascript/stylesheets/_colors.scss, and reuse it